pyvista/plotting/charts.py

```diff
@@ -1053,7 +1062,12 @@ def _resize(self):
         Resize this chart such that it always occupies the specified
         geometry (matching the specified location and size).
         """
-        r_w, r_h = self._renderer.GetSize()  # Alternatively: self.scene.GetViewWidth(), self.scene.GetViewHeight()
+        (
+            r_w,
+            r_h,
+        ) = (
+            self._renderer.GetSize()
+        )  # Alternatively: self.scene.GetViewWidth(), self.scene.GetViewHeight()
```

Comment:
Noooo :laughing: Perhaps if we put the comment after the expression on a new line?

Reply:
Would prefer having comment on a new line.